contact.html

```diff
@@ -1,22 +1,52 @@
 <!DOCTYPE html>
 <html lang = "en">
+	<!-- http://www.w3schools.com/html/html_layout.asp -->
+	
   <title>
   Poodle Security Training School | Contact Us
   </title>
-<body style="font-family:verdana;">
-<!-- http://www.w3schools.com/html/html_layout.asp -->
-<center>
-	<header><h1>Poodle Security Training School</h1></header>
+  <link rel="shortcut icon" href="http://www.hddevelop.com/images/pawprint.ico">
+  <meta charset="utf-8">
+	
+	
+	<style>
+	body{
+	  font-family: Verdana, sans-serif;
+	}
+	
+	.center{
+	  text-align: center;
+	}
+	
+	.bars{
+	  background-color: gray;
+	  color:white;
+	  padding: 25px 0px 25px 0px;
+	}
+	  
+	.content{
+	  padding: 25px 0px;
+	}
+	
+	p{
+	  padding: 0px 25px;
+	}
+  </style>
+	
+	
+<body>
+	<header class="center bars"><h1>Poodle Security Training School</h1></header>
 
-	<nav><a href="index.html">Homepage</a> | <a href="classes.html">Classes</a> | <a href="about.html">About Us</a> | Contact Us</nav>
-</center>
+	<nav class="center" style="padding: 10px 0px 0px 0px"><a href="index.html">Homepage</a> | <a href="classes.html">Classes</a> | <a href="about.html">About Us</a> | Contact Us</nav>
 
+<section class="content">
```

Comment:
I put the main text of the page in a ``<section>`` tag, instead of having it in the ``<body>``.

the reason behind this is because i wanted the main text to have some space added between the ``<nav>`` stuff and the ``<footer>`` stuff. so, I created the ``.content`` class, then used this ``<section>`` tag to only assign that class to this part. I'm pretty sure you could replace ``<section>`` with ``<div>``